ui-components/src/navigation/ProgressNav.tsx

```diff
@@ -26,10 +27,14 @@ const ProgressNavItem = (props: {
     )
 
   return (
-    <li className={`progress-nav__item ${bgColor}`}>
-      <a aria-disabled={bgColor === "is-disabled"} href={"#"}>
-        {srText}
+    <li className={`progress-nav__${props.style}-item ${bgColor}`}>
+      <a
+        aria-disabled={bgColor === "is-disabled"}
+        aria-current={bgColor === "is-active"}
+        href={"#"}
+      >
         {props.label}
+        {srText}
```

Comment:
This should be added before the label since at the moment it looks like this: `Household Current Step:`

Reply:
This change came from my interpretation of "Step state should add sr-only text of step state to the end of step title" in the issue, but feeling a bit unsure. Do you know if there is a different implementation of this comment?

Reply:
Oh I think the implementation is great, I would just switch lines 36 and 37

Reply:
Hmm I may be overthinking this one. I was interpreting that line to literally mean add the sr-only text to the end of the step title since the {srText} {props.label} order was previously in place. @slowbot do you mind clarifying what you intended in the issue?